superset-frontend/src/explore/constants.ts

```diff
@@ -136,4 +136,9 @@ export const TIME_FILTER_MAP = {
 
 // TODO: make this configurable per Superset installation
 export const DEFAULT_TIME_RANGE = 'No filter';
+// Default time range endpoints [start, end)
+export const DEFAULT_TIME_RANGE_ENDPOINTS: TimeRangeEndpoints = [
```

Comment:
Good fix! Could we have a way to get `DEFAULT_TIME_RANGE_ENDPOINTS` from the backend?

Reply:
This is fixed by SIP-15 #6360, are you sure want to provide an option allowing user customizing it? Besides, maybe we should create another issues, providing this as new feature instead of part of bugfix.